executor/runcexecutor/executor_linux.go

```diff
@@ -44,13 +44,10 @@ func (w *runcExecutor) exec(ctx context.Context, id, bundle string, specsProcess
 type runcCall func(ctx context.Context, started chan<- int, io runc.IO) error
 
 func (w *runcExecutor) callWithIO(ctx context.Context, id, bundle string, process executor.ProcessInfo, started func(), call runcCall) error {
-	runcProcess := &startingProcess{
-		ready: make(chan struct{}),
-	}
+	runcProcess, runcCtx, cancel := runcProcessHandle(ctx, id)
```

Comment:
Do we need both `cancel` and `Release()`? If the timing is different then `cancel` can still just be a method on `runcProcess` (`.Close()` or `.Stop()`).

Reply:
Yeah, the cancel was used to stop the signal/resize loops, allowing the waitgroup to exit cleanly.  I have stored the cancel in the procHandle and it will be called with a `runcProcess.Shutdown()` call now.